recipes/xilinx-runtime/meta.yaml

```
@@ -0,0 +1,85 @@
+{% set name = "xilinx-runtime" %}
+{% set version = "2.13.466" %}
+{% set quarter = "202210" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/Xilinx/XRT/archive/refs/tags/{{ quarter }}.{{ version }}.tar.gz
+  sha256: d73d8366f6f6d66bfdcb67e72135f559a4d95644c5416c27dea79c235953fd20
+  patches:
+    - patches/00_disable_werror.patch
+    - patches/01_xspi_std_array.patch
+    - patches/02_prid64_macro_usage.patch
+    - patches/03_icd.patch
+    - patches/04_no_static_linkage.patch
+    - patches/05_fix_python_rm_dkms_general_cmake.patch
+    - patches/06_fix_pkg_config.patch
+    - patches/07_no_version_for_dkms.patch
+    - patches/08_fix_cmake_install_prefix.patch
+    - patches/09_fix_ocl.patch
+    - patches/10_no_cloud_daemon.patch
+    - patches/11_fix_xbflash_install.patch
+    - patches/12_no_setup_script.patch
+    - patches/13_fix_appdebug_install.patch
+    - patches/14_remove_static_libs.patch
+
+build:
+  number: 0
+  skip: true  # [not linux]
+  ignore_run_exports_from:
+    - libdrm
+    - libcurl
+
```

```suggestion
  run_exports:
    - {{ pin_subpackage('xilinx-runtime'), max_pin='x') }}

```
Because you are shipping headers, you need a run_export which describes the ABI compatibility of this package for downstream packages that link against it. Please adjust the max_pin according to the compatibility guarantees of this library.

---

Thanks for pointing this our, this is fixed now, I am just following the lead from libprotobuf and json-c. Resolving this.